tests/test_updater_old.py

```diff
@@ -21,7 +21,7 @@
   See LICENSE-MIT OR LICENSE for licensing information.
 
 <Purpose>
-  'test_updater.py' provides a collection of methods that test the public /
+  'test_updater.py_old' provides a collection of methods that test the public /
```

Comment:
Is this the coding equivalent of the french Verlan? :P Given the short remaining life time of this doc, I say we leave this as is.